core/extensions.rs

```diff
@@ -586,6 +586,51 @@ impl Extension {
     self.esm_entry_point
   }
 
+  pub fn get_ops_virtual_module(&self) -> Option<ExtensionFileSource> {
+    // TODO(bartlomieju): not great to hardcode `core` here.
+    // TODO(bartlomieju): checking for files beings empty is a bit error
+    // prone, but we don't store information if we're snapshotting or not
+    // in extension.
+    eprintln!("get ops virtual module {:#?}", self.name);
+    if self.name == "core"
+      || self.ops.is_empty()
+      || (self.js_files.is_empty()
+        && self.esm_files.is_empty()
+        && self.esm_entry_point.is_none())
+    {
+      return None;
+    }
+
+    let op_names: Vec<_> = self
+      .ops
+      .iter()
+      .filter(|o| o.enabled)
+      .map(|o| o.name)
+      .collect();
+
+    let mut output = Vec::with_capacity(op_names.len() * 2 + 4);
+    output.push("const {".to_string());
+
+    for op_name in &op_names {
+      output.push(format!("  {},", op_name));
+    }
+
+    output.push("} = Deno.core.ensureFastOps(true);".to_string());
```

Comment:
Ideally we could remove this `true`, but I think we're gonna face many crashes before we migrate all the code to use the new system.

Reply:
Also, it's unclear what would be interaction with enabled/disabled ops.

Reply:
Eg. if I remove the `true` I get failures in `testing/unit/ops_buffer_test.ts` which also uses `ensureFastOps(true)`. But I can use `import { ... } from "ext:/...` there because it's a test file that will fail a check in module loader. 

Ideas?

Reply:
> Also, it's unclear what would be interaction with enabled/disabled ops.

I think we need to throw from disabled ops as there's no way to have an optional import is there?



> Eg. if I remove the `true` I get failures in `testing/unit/ops_buffer_test.ts` which also uses `ensureFastOps(true)`. But I can use `import { ... } from "ext:/...` there because it's a test file that will fail a check in module loader.
> 
> Ideas?

I think we'll just need to reexport here:

What about this?

```
import * as ops from "ext:...";
globalThis.testOps = ops;
```

Reply:
This will become a non-issue once we land https://github.com/denoland/deno_core/pull/456